lib/array-store.js

```diff
@@ -27,7 +27,7 @@ ArrayStore.prototype.read = function(id) {
   var object = _.find(this.data, function(_object) {
     return String(_object.id) === String(id);
   });
-  return Promise.resolve(object);
+  return Promise.resolve(_.clone(object));
```

Comment:
If we don't return a clone the object can be mutated outside of the store, which is a behavior different from the $fh.db version.
It actually introduces a bug in -user otherwise.